react/modules/module04/README.md

```diff
@@ -1,47 +1,45 @@
-# Неделя 4
+# Week 4
 
 ## React. Hooks
 
 
 
 
-!! Задание выполняется в школьном приватном репозитории !!
-Нужно выполнять задание на базе предыдущего
+!! The task must be done in the school’s private repository !! 
+Do the task on the basis of the previous one
 
 
 
 
-Что должно быть сделано:
+What must be done:
 
 
 
-1) Создать отдельную ветку для этого задания
-2) Переписать основную страницу на хуки сохранив функционал. Все тесты по итогу должны работать
-3) Переписать форму на React Hook Form, сохранив функционал. Все тесты по итогу должны работать
+1) Create a separate branch for this task
+2) Saving functionality, rewrite the main page with the hooks. All the test should be working in the end
+3) Saving functionality, rewrite the form with React Hook Form. All the test should be working in the end
 
+### Score
 
-### Оценка
 
+The task will be evaluated by a mentor during the first 6 weeks. To make the evaluation more convenient, create a Pull Request. (DO NOT MERGE).
```

Comment:
Also, we can be more polite :)
To make the evaluation more convenient, please, create a Pull Request.